lib/dl_core/dl_core/connection_executors/adapters/async_adapters_aiohttp.py

```diff
@@ -88,12 +89,15 @@ def get_session_timeout(self) -> ClientTimeout:
     def get_session_auth(self) -> Optional[BasicAuth]:
         return None
 
-    def get_session_headers(self) -> Dict[str, str]:
```

Comment:
let's remove the import as well (one day we'll enable stricter CI checks, I swear)

Reply:
Oh, sorry, I already did it, but forgot to add to the commit